src/runtime/virtcontainers/virtiofsd.go

```diff
@@ -216,7 +218,8 @@ func (v *virtiofsd) kill(ctx context.Context) (err error) {
 	defer span.End()
 
 	if v.PID == 0 {
-		return errors.New("invalid virtiofsd PID(0)")
+		v.Logger().Warnf("invalid virtiofsd: %v", v.PID)
```

Comment:
Since v.PID == 0 , need to log its value?

Reply:
Yes, it's better to log the invalid value and let users know what's wrong with it.

Reply:
Nit: Ideally, we'd use a log field. Something like:

```go
v.Logger().WithField("invalid-virtiofsd-pid", v.PID).Warn("cannot kill virtiofsd")
```